js/dispatch.ts

```diff
@@ -22,7 +30,19 @@ export function asyncMsgFromRust(opId: number, ui8: Uint8Array): void {
     case OP_READ:
       minimal.asyncMsgFromRust(opId, ui8);
       break;
+    case OP_EXIT:
+    case OP_IS_TTY:
+    case OP_ENV:
+    case OP_EXEC_PATH:
     case OP_UTIME:
+    case OP_APPLY_SOURCE_MAP:
+    case OP_FORMAT_ERROR:
+    case OP_CACHE:
+    case OP_FETCH_SOURCE_FILE:
+    case OP_OPEN:
+    case OP_CLOSE:
+    case OP_SEEK:
+    case OP_FETCH:
```

Comment:
Only put async ops in this list - that will induce an error through the default case below if we unexpectedly get an async response for a sync op.

Reply:
Ah, makes sense! Will update